Library/Homebrew/test/cache_store_spec.rb

```diff
@@ -89,6 +89,34 @@
     end
   end
 
+  describe "#clear" do
+    context "with a database created" do
+      let(:db) { instance_double(Hash, "db", :[] => { foo: "bar" }) }
+
+      before do
+        allow(sample_db).to receive_messages(created?: true, db:)
+      end
+
+      it "clears values in `CacheStoreDatabase`" do
+        expect(db).to receive(:clear)
+        sample_db.clear
```

Comment:
This sort of test is pretty much just checking the code exactly mirrors the test and will break if there's any meaningful refactoring done there. I'm not strongly 👎🏻 on it being there but: I'm not sure what value these new tests add so may be worth removing unless others feel very strongly otherwise (in which case: I think another test would be better)

Reply:
removed.